### PR TITLE
Merge `main` to `release-1.11.0`

### PR DESCRIPTION
**Pull request by rishabhatdell**
# Description
This PR merge the main changes to release-1.11.0 branch

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1221|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -21,5 +21,6 @@
 # Sharon Toll (sharont58)
 # Shayna Finocchiaro (shaynafinocchiaro)
 # Shefali Malhotra (shefali-malhotra)
+# Shefali Malhotra (shefali-malhotra)
```
Comment (shanmydell) [inline .github/CODEOWNERS:24]: there is two entries

**Review by shanmydell — CHANGES_REQUESTED** [review]
Pls address the comments

File: content/docs/authorization/v1.x GA/Backup and Restore/rpm/_index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ description: >
 ---
 
 {{% pageinfo color="primary" %}}
-The CSM Authorization RPM is no longer actively maintained or supported. It will be deprecated in CSM 2.0. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
+The CSM Authorization RPM is no longer actively maintained or supported. It will be deprecated in a future release. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
```
Comment (shanmydell) [inline content/docs/authorization/v1.x GA/Backup and Restore/rpm/_index.md:9]: @gallacher : Do we have a timeline for deprecation?

File: content/docs/csidriver/release/powerflex.md [hunk 3]
```diff
@@ -25,6 +27,7 @@ description: Release notes for PowerFlex CSI driver
 - [#1163 - [BUG]: Resource quota bypass](https://github.com/dell/csm/issues/1163)
 - [#1174 - [BUG]: Kubelet Configuration Directory setting should not have a comment about default value being None](https://github.com/dell/csm/issues/1174)
 - [#1210 - [BUG]: Helm deployment of PowerFlex driver is failing](https://github.com/dell/csm/issues/1210)
+- [#1210 - [BUG]: Helm deployment of PowerFlex driver is failing](https://github.com/dell/csm/issues/1210)
```
Comment (shanmydell) [inline content/docs/csidriver/release/powerflex.md:30]: 29 and 30 line is repeated

File: content/docs/csidriver/release/powerscale.md [hunk 4]
```diff
@@ -41,3 +43,4 @@ description: Release notes for PowerScale CSI driver
 ### Note
 
 - Support for Kubernetes alpha features like Volume Health Monitoring will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.
+- Support for Kubernetes alpha features like Volume Health Monitoring will not be available in Openshift environment as Openshift doesn't support enabling of alpha features for Production Grade clusters.
```
Comment (shanmydell) [inline content/docs/csidriver/release/powerscale.md:46]: 45 and 46 line is repeated

Comment (rishabhatdell) [reply content/docs/csidriver/release/powerscale.md]: fixed

File: content/docs/csidriver/release/powerflex.md (hunk 3 above)
Comment (rishabhatdell) [reply content/docs/csidriver/release/powerflex.md]: fixed

File: .github/CODEOWNERS (hunk 1 above)
Comment (rishabhatdell) [reply .github/CODEOWNERS]: fixed

**Review by AkshaySainiDell — DISMISSED** [review]
LGTM

**Review by suryagupta4 — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**